.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-	"presets": ["es2015", "react"],
-	"plugins": ["transform-object-rest-spread", "transform-es2015-modules-commonjs"]
+	"presets": ["es2015", "stage-0", "react"],
```

Comment:
I use 'babel-preset-stage-0' replace 'babel-plugin-transform-object-rest-spread',
this is not necessary, just in case forgot to add transform plugins.